scripts/generate-artifacts.js

```diff
@@ -19,7 +19,8 @@ const facets = [
     "FraudProofFacet",
     "JoinChannelFacet",
     "StateChannelCommon",
-    "StateSnapshotFacet"
+    "StateSnapshotFacet",
+    "LocalDiamond"
```

Comment:
We have too many places in the code that we hardcode stuff either in some records mapping or like this or even ethers types and it's becoming too much to remember to update/manage - for instance even here we forgot to add `DisputeVerificationFacet` that was the result of splitting `DisputeManagerFacet` maybe other stuff is missing too 

Not sure what's the best way of addressing this - we could write a script for this that does some regex on the generated artifacts to extract all `*Facet` - that would automate this

Add some trello card for this and generally thinking how to NOT have hard coded stuff in our code that we have to remember to update